packages/extester-runner/src/ui-test/03_extersterTreeProvider.test.ts

```diff
@@ -0,0 +1,442 @@
+		assert.strictEqual(firstItLabel.trim(), EXPECTED_TEST);
+	});
+
+	/**
+	 * Verifies parsing of a test file with multiple root describe blocks
+	 * Tests that multiple root-level describe blocks are correctly displayed in the tree view
+	 */
+	it('multipleRootDescribe.test.ts', async function () {
+		const EXPECTED_DESCRIBE_1 = 'Root suite 1';
+		const EXPECTED_TEST_1 = 'Root suite 1 test 1';
+		const EXPECTED_DESCRIBE_2 = 'Root suite 2';
+		const EXPECTED_TEST_2 = 'Root suite 2 test 1';
+
+		// Setup test view
+		const runnerView = await (await new ActivityBar().getViewControl(EXTESTER_RUNNER))?.openView();
+		assert.ok(runnerView, 'Runner view not found');
+
+		const content = await runnerView.getContent();
+		assert.ok(content, 'Content not found');
+
+		const sections = await content.getSections();
+		assert.ok(sections?.length > 0, 'No sections found');
+
+		const tree = sections[0] as DefaultTreeSection;
+
+		// Verify parser folder exists
+		const items = await tree.getVisibleItems();
+		const labels = await Promise.all(items.map((item) => item.getLabel()));
+		expect(labels).contains(PARSER_FOLDER);
+
+		// Navigate to test file
+		await tree.openItem(PARSER_FOLDER);
+		const item = await tree.findItem(MULTIPLE_ROOT_DESCRIBES);
+		await item?.expand();
+
+		// Verify describe blocks
+		const describes = await item?.getChildren();
+		assert.ok(describes?.length === 2, `${MULTIPLE_ROOT_DESCRIBES} should have exactly two describes`);
+
+		// first describe
+		const firstDescribe = describes[0];
+		const firstDescribeLabel = await firstDescribe.getLabel();
+		assert.strictEqual(firstDescribeLabel.trim(), EXPECTED_DESCRIBE_1);
+
+		// first describe first test
+		const firstDesribeIts = await firstDescribe.getChildren();
+		assert.ok(firstDesribeIts?.length === 1, 'First describe should have exactly one it');
+
+		const firstDesribeFirstIt = firstDesribeIts[0];
+		const firstDesribeFirstItLabel = await firstDesribeFirstIt.getLabel();
+		assert.strictEqual(firstDesribeFirstItLabel.trim(), EXPECTED_TEST_1);
+
+		// second describe
+		const secondDescribe = describes[1];
+		const secondDescribeLabel = await secondDescribe.getLabel();
+		assert.strictEqual(secondDescribeLabel.trim(), EXPECTED_DESCRIBE_2);
+
+		// second describe first test
+		const secondDescribeIts = await secondDescribe.getChildren();
+		assert.ok(secondDescribeIts?.length === 1, 'Second describe should have exactly one it');
+
+		const secondDescribeFirstIt = secondDescribeIts[0];
+		const secondDescribeFirstItLabel = await secondDescribeFirstIt.getLabel();
+		assert.strictEqual(secondDescribeFirstItLabel.trim(), EXPECTED_TEST_2);
+	});
+
+	/**
+	 * Verifies parsing of a complex test file with nested describe blocks and various modifiers
+	 * Tests that complex test structures are correctly displayed in the tree view
+	 *
+	 * @note This test is currently skipped as the content is too large to fit on the test screen
+	 */
+	it.skip('complexFile.test.ts', async function () {
```

Comment:
what is a limitation for a too large content? a known issue or something new?

Reply:
Content of Tree won't fit on screen (it's to long) so method getVisibleItems won't get whole content.